setup.py

```diff
@@ -1,3 +1,4 @@
+##! /usr/bin/env python
```

Comment:
Was this supposed to be a valid shebang?

Reply:
(typo)